modules/dev2plus/hw_01/test/Tree.test.js

```diff
@@ -0,0 +1,37 @@
+const Node = require("../src/Node/Node");
+const { Tree } = require("../src/Tree/Tree");
+
+const mockDataSet = {
+  value: {
+    name: "James Smith",
+    salary: 15,
+  },
+  leftChild: {
+    value: {
+      name: "John Johnson",
+      salary: 10,
+    },
+    leftChild: {
+      value: {
+        name: "Mary Williams",
+        salary: 7,
+      },
+    },
+  },
+  rightChild: {
+    value: {
+      name: "Richard Davis",
+      salary: 13,
+    },
+  },
+};
+
+test("Tree obj should be an extension of the Tree", () => {
+  const ST = new Tree();
+  expect(ST instanceof Tree).toBe(true);
+});
+
+test("Tree should not have an insert method", () => {
```

Comment:
this is wrong, a tree must have an insert method, how do you add nodes without it?